documentation/Bundling.md

```diff
@@ -223,7 +223,6 @@ export default {
     cjs({
```

Comment:
As Will mentioned in the issue, this isn't even needed when using rollup 2.x but it will error using rollup 1.x - I figured better to keep it and get a warning if you're on rollup 2.x then remove it and get an error if you're on rollup 1.x (which we are using elsewhere)